jobs/package-dockertested/Jenkinsfile

```diff
@@ -92,21 +92,106 @@ node('buildvm-devops') {
 			stage ('Build an Origin release') {
 				sh """{
 cat <<EOF
-sudo su origin
 set -o errexit -o nounset -o pipefail -o xtrace
 cd /data/src/github.com/openshift/origin
 hack/build-base-images.sh
-make release
+OS_BUILD_ENV_PRESERVE=_output/local hack/env OS_ONLY_BUILD_PLATFORMS='linux/amd64' hack/build-rpm-release.sh
+sudo systemctl restart docker
```

Comment:
I thought there was a commit message in there somewhere with the explanation ... hmmm ... yes we can add it